code/src/pages/Actor.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { BackButton } from 'BackButton'
+
+export const Actor = () => {
+  const { actorID } = useParams();
+  const [actor, setActor] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/person/${actorID}?api_key=303d50a32fc0419fb55796d006e5d6c2&language=en-US`)
+    .then(res => {
+      if (res.ok) {
+        return res.json();
+      } else {
+        throw new Error('404');
+      }
+    })
+   .then((json) => {
+     setActor(json);
+   })
+   .catch(() => {
+    window.location.assign('/404');
+  });
+  },[actorID])
+
+return (
+  <section className="actor-main-content">
+    <header className="app-header"> <BackButton text="Back to movie"></BackButton></header>
```

Comment:
No need for an opening and closing tag when importing a component, it should be a self closing tag